server/src/main/java/com/cloud/network/router/NicProfileHelperImpl.java

```diff
@@ -118,7 +118,7 @@ public NicProfile createGuestNicProfileForVpcRouter(final RouterDeploymentDefini
         final NicProfile guestNic = new NicProfile();
 
         if (vpcRouterDeploymentDefinition.isRedundant()) {
-            guestNic.setIPv4Address(_ipAddrMgr.acquireGuestIpAddress(guestNetwork, null));
+            guestNic.setIPv4Address(_ipAddrMgr.acquireFirstGuestIpAddress(guestNetwork));
```

Comment:
@ArthurHlt should acquireGuestIpAddressForVrouterRedundant be used here  as well?